calico-enterprise_versioned_docs/version-3.15/variables.js

```diff
@@ -1,12 +1,12 @@
 const releases = require('./releases.json');
 
 const variables = {
-  releaseTitle: 'v3.15.1',
+  releaseTitle: 'v3.15.2',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.15',
-  baseUrl: '/calico-enterprise/latest',
-  filesUrl: 'https://downloads.tigera.io/ee/v3.15.1',
+  baseUrl: '/calico-enterprise/3.15',
```

Comment:
s/3.15/latest!

Reply:
I misunderstood...I thought you meant to replace it, not add "latest"